test/app/components/content_for_component.rb

```diff
@@ -1,5 +1,12 @@
 # frozen_string_literal: true
 
 class ContentForComponent < ActionView::Component::Base
-  def initialize(*); end
+  validates :content, :title, :footer, presence: true
+
+  attr_reader :content, :title, :footer
```

Comment:
Nit: the `content` reader should already be defined in `Base`, and not here.
```suggestion
  attr_reader :title, :footer
```

Reply:
fixed